packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -539,4 +620,15 @@ class HeroController extends NavigatorObserver {
   void _handleFlightEnded(_HeroFlight flight) {
     _flights.remove(flight.manifest.tag);
   }
+
+  static final HeroFlightShuttleBuilder _defaultHeroFlightShuttleBuilder = (
+    BuildContext flightContext,
+    Animation<double> animation,
+    HeroFlightDirection flightDirection,
+    BuildContext fromHeroContext,
+    BuildContext toHeroContext,
+  ) {
+    final Hero toHero = toHeroContext.widget;
+    return toHero.child;
+  };
```

Comment:
we should make sure this is exactly what we used to do before (i can't tell from the code if this is a change from what we did before, but the documentation you added earlier says that it is).

Reply:
(i find it concerning that no tests broke as a result of this change)